ts/components/conversation/message/message-content/MessageContentWithStatus.tsx

```diff
@@ -151,7 +151,7 @@ export const MessageContentWithStatuses = (props: Props) => {
           />
         )}
       </ExpirableReadableMessage>
-      {enableReactions && (
+      {!isDetailView && enableReactions ? (
```

Comment:
might be useful to pass down this boolean with a context at some point, let's keep it in mind

Reply:
👍